1.1/reqs.ttl

```diff
@@ -1,980 +1,1157 @@
+    prov1:wasDerivedFrom conf10core:sparql-protocol ;
+.
+
+conf11core:spatial-object-collection-class
+    a spec:ConformanceTest ;
+    spec:testPurpose "check conformance with this requirement"@en ;
+    spec:testType spec:Capabilities ;
+    skos:definition "Verify that queries involving geo:SpatialObjectCollection return the correct result on a test dataset"@en ;
+    skos:prefLabel "GeoSPARQL 1.1 Conformance Test: Spatial Object Collection Class"@en ;
+.
+
+conf11core:spatial-object-properties
+    a spec:ConformanceTest ;
+    spec:testPurpose "check conformance with this requirement"@en ;
+    spec:testType spec:Capabilities ;
+    rdfs:seeAlso reqs11core:spatial-object-properties ;
+    skos:definition "Verify that queries involving these properties return the correct result for a test dataset"@en ;
+    skos:prefLabel "GeoSPARQL 1.1 Conformance Test: Spatial Object Properties"@en ;
+.
+
+conf11gx:dggs-literal
+    a spec:ConformanceTest ;
+    spec:testPurpose "check conformance with this requirement"@en ;
+    spec:testType spec:Capabilities ;
+    rdfs:seeAlso reqs11gx:dggs-literal ;
+    skos:definition "Verify that queries do not use use this datatype but instead use specializations of it"@en ;
+    skos:prefLabel "GeoSPARQL 1.1 Conformance Test: DGGS Literal"@en ;
+.
+
+conf11gx:dggs-literal-empty
+    a spec:ConformanceTest ;
+    spec:testPurpose "check conformance with this requirement"@en ;
+    spec:testType spec:Capabilities ;
+    rdfs:seeAlso reqs11gx:dggs-literal-empty ;
+    skos:definition "Verify that queries involving empty geo:dggsLiteral values return the correct result for a test dataset"@en ;
+    skos:prefLabel "GeoSPARQL 1.1 Conformance Test: DGGS Literal Empty"@en ;
+.
+
+conf11gx:feature-properties
+    a spec:ConformanceTest ;
+    spec:testPurpose "check conformance with this requirement"@en ;
+    spec:testType spec:Capabilities ;
+    rdfs:seeAlso reqs11gx:feature-properties ;
+    skos:definition "Verify that queries involving these properties return the correct result for a test dataset"@en ;
+    skos:prefLabel "GeoSPARQL 1.1 Conformance Test: Feature Properties"@en ;
+.
+
+conf11gx:geojson-literal
+    a spec:ConformanceTest ;
+    spec:testPurpose "check conformance with this requirement"@en ;
+    spec:testType spec:Capabilities ;
+    rdfs:seeAlso reqs11gx:geojson-literal ;
+    skos:definition "Verify that queries involving geo:geoJSONLiteral values return the correct result for a test dataset"@en ;
+    skos:prefLabel "GeoSPARQL 1.1 Conformance Test: GeoJSON Literal"@en ;
+.
+
+conf11gx:geojson-literal-empty
+    a spec:ConformanceTest ;
+    spec:testPurpose "check conformance with this requirement"@en ;
+    spec:testType spec:Capabilities ;
+    rdfs:seeAlso reqs11gx:geojson-literal-empty ;
+    skos:definition "Verify that queries involving geo:kmlLiteral values without an explicit encoded SRS IRI return the correct result for a test dataset"@en ;

```

Comment:
Should be "Verify that queries involving empty geo:geoJSONLiteral values return the correct result for a test dataset"